ui/src/session.ts

```diff
@@ -151,6 +151,15 @@ export const updateAppearance = (appearance: Appearance): void =>
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     settings: { ...get(settings), appearance },
   });
+
+export const dismissRemoteHelperHint = (): void => {
+  updateAppearance({
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
What does this exception do?

Reply:
It's a leftover from our packaging upgrade. Should be tackled in: https://github.com/radicle-dev/radicle-upstream/issues/696.

Read more about it [here](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unsafe-member-access.md.).